client/benches/tps/lib.rs

```diff
@@ -67,6 +70,9 @@ impl Config {
             let handle = unit.spawn_event_counter();
             handles.push(handle)
         }
+        // Sleep to let the blocks produced by units to be committed on all peers
+        thread::sleep(core::time::Duration::from_secs(1));
```

Comment:
Yes, this seems to be needed because `unit.ready()` guarantees no more than that the blocks are committed on the unit itself